private/src/avb_streamhandler/IasAvbStreamHandler.cpp

```diff
@@ -435,7 +435,21 @@ IasAvbProcessingResult IasAvbStreamHandler::init( const std::string& configName,
         // TODO: Add a config option to receive user input to disable avb_watchdog at runtime.
         //       Otherwise, would not be able to start by executing the demo_app normally
         //       since WATCHDOG_USEC is set by systemctl.
```

Comment:
This commentary is funny - why not simply do something like `WATCHDOG_USEC=5000 ./avb_streamhandle_demo`? I know it was before your change, but I don't think it makes sense.

Reply:
Yea, I've gone through that comment several times, but left it as is as I wanted to avoid triggering a bigger validation effort.